### ISSUE TITLE
Error when creating new menu item: attempt to read property "language"; Mega Menu button no longer showing

### ISSUE THREAD

**Issue by jlilio** [templaza/astroid-framework#260]
### Discussed in https://github.com/templaza/astroid-framework/discussions/259

<div type='discussions-op-text'>

<sup>Originally posted by **jlilio** December  8, 2022</sup>
Development Environment:
- XAMPP for Windows 8.0.23
- Joomla 4.2.5
- Astroid Framework Version: 2.6.3
- Template Version: 2.6.3 (One)

Error when creating new menu on the back end administration:
Warning: Attempt to read property "language" on null in <path>\libraries\astroid\framework\layouts\fields\astroidmenuoptions.php on line 62

Effect: When creating a new menu, Astroid Menu Options > Mega Menu button is no longer showing. Downgrading to framework 2.6.2 did not correct the error. All previously created menus are fine (i.e., no error message pertaining to language and mega menu is still available).

This may be a bug. Any ideas?

![20221208-issue-astroidmenuoptions-line-62-attempt-to-read-property-language-error-megamenu-missing](https://user-images.githubusercontent.com/6914851/206367779-ce25b1b8-67f0-4fe1-9f8e-09b446dc6db5.png)
</div>

**Comment by sonvnn** [comment]
Hi @jlilio ,
I fixed this issue in https://github.com/templaza/astroid-framework/commit/f22141d57b04e8752235c85a7ca4dea68df13d6d

Please help me download `developer branch` and upload to your site. https://github.com/templaza/astroid-framework/archive/refs/heads/developer.zip

Please let me know if this issue still happen again.

Best Regards,
Sonny

**Comment by jlilio** [comment]
Hi @sonvnn

Thank you for this, Sonny. I have tested and confirm that the dev branch has fixed the issue. The error message is gone and any new menu created will show the mega menu already.

Note that this is applicable for any newly created menu items only. It takes effect after creating then saving/closing the new menu. After opening, the mega menu appears. Any existing menu items don't get fixed even after editing and saving them (i.e., the mega menu still doesn't appear).

I have tested the dev branch on both Joomla 4.2.5 and 4.2.6-rc1.

Thanks again.

Best regards

Julian

> Hi @jlilio , I fixed this issue in [f22141d](https://github.com/templaza/astroid-framework/commit/f22141d57b04e8752235c85a7ca4dea68df13d6d)
> 
> Please help me download `developer branch` and upload to your site. https://github.com/templaza/astroid-framework/archive/refs/heads/developer.zip
> 
> Please let me know if this issue still happen again.
> 
> Best Regards, Sonny
